### ISSUE TITLE
Panorama: Not working in Explore Dashboard

### ISSUE THREAD

**Issue by AlecBlance** [AprilSylph/XKit-Rewritten#1041]
### Platform

Windows 11

### Browser

Chrome 111.0.5563.147 

### Addon version

master (0.21.4)

### Details

This happened because of the added parent div `mainColumContainer` that contains the `main` element. Originally, this isn't the case in Home dashboard.

```html
<!-- In Home -->
<div class="main">
    <main>  
```

```html
<!-- In Explore -->
<div class="main">
    <div class="mainColumContainer">
        <main>
```

The selector isn't targeting the `main` element since it looks for it as a child of `:first-child`. 

https://github.com/AprilSylph/XKit-Rewritten/blob/c2c34d71ab48bf15ee270ab13f162f9d1f2d6bde/src/scripts/panorama.js#L23-L25

![image](https://user-images.githubusercontent.com/17378596/228135025-21156f73-6fbd-4972-aaf9-4698dbb8a3d2.png)

**Comment by marcustyphoon** [comment]
I can't remember: does #845 solve this? I believe it did not, since I had some trouble with that page whose details I no longer recall. But that might have been a different page.

**Comment by AlecBlance** [comment]
Yes, I think it did not, but you can include the fix in your PR. 😄

**Comment by marcustyphoon** [comment]
I wasn't able to get it to work, I believe! If someone does pick this up they should be ready to solve any merge conflicts with that PR, though.
